spec/features/1_basic_spec.rb

```diff
@@ -0,0 +1,129 @@
+require "rails_helper"
+
+describe "The /movies page" do
+  it "can be visited", points: 1 do
+    visit "/movies"
+
+    expect(page.status_code).to be(200),
+      "Expected to visit /movies successfully."
+  end
+
+  it "has a link to add a movie", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_link('Add a new movie', href: "/movies/new"),
+      "Expected /movies to have an 'Add a new movie' link to '/movies/new'."
+  end
+
+  it "has a bootstrap navbar", points: 1 do
```

Comment:
Since you're testing for a specific type of bootstrap navbar, it might be nice to specify that here. Maybe something like this:

```suggestion
  it "has a large, light-themed bootstrap navbar", points: 1 do
```

What do you think?

Reply:
I agree, changed this